fontforge/python.c

```diff
@@ -8620,9 +8686,21 @@ return( NULL );
 	format=2;
 
     if ( format!=-1 ) {
-	if ( !PyArg_ParseTuple(args,"O|ii",&foo,&pixels,&bits) ) {
-	    free(locfilename);
-return( NULL );
+	if ( obj1!=NULL ) {
```

Comment:
How is this equivalent to before? i.e. why did it parse out `bar` then never use it? A bug?

Reply:
Before, `foo` and `bar` were just to make the initial `ParseTuple` work when used with both optional arguments to get the filename. Then that would be parsed and `ParseTuple` would be called again on `args` to extract the particular values depending on whether the format is raster or vector. 

Given that the layer can be specified as an int or a string and @ctrlcctrlv broke out the code to figure that out from a PyObject it seemed easier to just grab the two values as objects and examine them that way.